app/_landing_pages/ai-gateway.yaml

```
@@ -349,6 +410,58 @@ rows:
           config:
             slug: ai-response-transformer
 
+
+  - header:
+      type: h3
+      text: "Automated RAG"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    LLMs are only as reliable as the data they can access. When faced with incomplete information, they often produce confident yet incorrect responses—known as “hallucinations.” These hallucinations occur when LLMs lack the necessary domain knowledge.To address this, developers use the **Retrieval-augmented Generation (RAG)** approach, which enriches models with relevant data pulled from vector databases.
+
+                    While standard RAG workflows are resource-heavy, as they require teams to generate embeddings and manually curate them in vector databases, Kong’s **AI RAG Injector** plugin automates this entire process. Instead of embedding RAG logic into every application individually, platform teams can inject vetted data into prompts directly at the gateway layer without any manual interventions.
+  - columns:
+      - blocks:
+        - type: plugin
+          config:
+            slug: ai-rag-injector
+
+  - header:
+      type: h3
+      text: "Load balancing"
+    columns:
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    AI Gateway’s load balancer routes requests across AI models to optimize for speed, cost, and reliability. It supports algorithms like consistent hashing, lowest-latency, usage-based, round-robin, and semantic matching, with built-in retries and fallback for resilience {% new_in 3.10%}. The balancer dynamically selects models based on real-time performance and prompt relevance, and works across mixed environments including OpenAI, Mistral, and Llama models.
+
+  - columns:
+      - blocks:
+        - type: card
+          config:
+            title: Load balancing
+            description: Learn about the load balancing algorithms available for AI Gateway.
+            icon: /assets/icons/load-balance.svg
+            cta:
+              url: /gateway/load-balancing/#load-balancing-algorithms-for-ai-gateway
```

```suggestion
              url: /ai-gateway/load-balancing/
```
I think it's better to link to your new page here, it's more relevant.